storage/mocks/storage.go

```diff
@@ -600,6 +600,21 @@ func (mr *MockEventsMockRecorder) ByBlockIDTransactionID(arg0, arg1 interface{})
 	return mr.mock.ctrl.RecordCallWithMethodType(mr.mock, "ByBlockIDTransactionID", reflect.TypeOf((*MockEvents)(nil).ByBlockIDTransactionID), arg0, arg1)
 }
 
+// ByBlockIDTransactionIndex mocks base method
```

Comment:
any reason for having to mocks files on under mocks and one under mock ?

Reply:
Two different mocking libraries maybe?